src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.dev.cinema.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@ComponentScan(basePackages = {"com.dev.cinema"})
```

Comment:
it's better to set specific packages (service, dao, etc), with general package you may receive problems with two configuration files, if they overlap

Reply:
ok. done